### PR TITLE
Adds missing end tag in code example for grep

### PR DESCRIPTION
**Pull request by froschdesign**
|    Q          |   A
|-------------- | ------
| Bugfix        | yes

### Description

docs laminas dev_migration_faq_

https://docs.laminas.dev/migration/faq/#migrating-with-a-zendframework-zendframework-package

### CODE REVIEW

File: migration/faq/index.html [hunk 1]
```diff
@@ -316,7 +316,7 @@ <h3>Install Individual Packages</h3>
     dependencies you have compiled. For each package in that list, run:
 </p>
 
-<pre class="codehilite"><code class="language-bash" data-trim>
+<pre class="codehilite"><code class="language-bash">
```
Comment (michalbundyra) [inline migration/faq/index.html]: We are missing closing `</code>` tag.

Comment (froschdesign) [reply migration/faq/index.html]: See: https://github.com/laminas/laminas.github.io/pull/58/commits/387e40201ee25bb0be4ee475891588422a0f02e2#diff-7e10905ce26eea322d830db37c66e7a3R273

Comment (michalbundyra) [reply migration/faq/index.html]: I am confused, this is the other place...

Comment (froschdesign) [reply migration/faq/index.html]: I was also confused but it is fixed now.

**Comment by michalbundyra** [comment]
Thanks, @froschdesign!

**Pull request merged**